docs/zh/advanced/cookbook/extending-a-theme.md

```diff
@@ -19,12 +22,38 @@ module.exports = {
 }
 ```
 
+  </CodeGroupItem>
+
+  <CodeGroupItem title="TS">
+
+```ts
+import { path } from '@vuepress/utils'
+import { ThemeObject } from '@vuepress/core'
+
+export default {
+  // 你的主题
+  name: 'vuepress-theme-foo',
+  // 要继承的父主题
+  extends: 'vuepress-theme-bar',
+  // 覆盖父主题的布局
+  layouts: {
+    Layout: path.resolve(__dirname, 'layouts/Layout.vue'),
+  },
+} as ThemeObject
```

Comment:
Should be consistent with the later snippet, i.e. `const fooTheme ...`

Reply:
Fixed.